README.md

```diff
@@ -1,2 +1,28 @@
+Status: Currently in draft and non-binding. 
+
 # Argo Conformance Program
-A standard to ensure interoperability, protect users, and prevent surprises when using commercial offerings with Argo in the name. Currently in draft and non-binding. 
++The Argo community continues to grow dramatically – adding new users, new contributors, and new capabilities. The Argo Conformance Program creates incentives and requirements designed to help the ecosystem grow in a sustainable and user-friendly way. As a diverse set of tools grows around Argo, users can trust that products certified as compliant will interoperate with each other and be up-to-date with the latest open source releases. This ensures a consistent user experience between products and flexibility to choose between vendors.
+
+When users use an Argo certified offering they should be able to trust that offering to work with the open source versions of Argo and that configuration designed for Argo will always work.
+
+Software conformance ensures predictability and helps protect the Argo brand. 
+
+# Goals
++* Guarantee a consistent and predictable user experience
++* Guarantee interoperability, including API compatibility
++* Guarantee conformant products are up-to-date with latest Argo releases 
++* Ensure contributions from commercial partners
```

Comment:
Do we want to use something softer, like "engagement" over contributions? As a goal, engagement feels better, then we can specify in the requirements exactly what it means

Reply:
I'm good with that @hblixt. @edlee2121 made a good point when I talked to him today about a race for people to get maintainers in. I think we can work it out in the requirements but I'm fine with either word.

Reply:
I don't really feel strongly either way. It just felt a little more appropriate for a goal to use engagement over contributions, which seemed a bit more of a requirement.

Reply:
Debated this in committee with @alexec @wanghong230 and @hblixt decided to move forward with current language. May revisit if someone wants conformance without maintainers.